packages/web-app-admin-settings/src/views/Users.vue

```diff
@@ -219,11 +219,12 @@ export default defineComponent({
     watch(selectedUserIds, async () => {
       sideBarLoading.value = true
       // Load additional user data
-      const requests = []
-      unref(selectedUsers).forEach((user) => {
-        requests.push(loadAdditionalUserDataTask.perform(user))
-      })
-      await Promise.all(requests)
+      const requests = unref(selectedUsers).map((user) => loadAdditionalUserDataTask.perform(user))
+      const results = await Promise.allSettled<Array<unknown>>(requests)
+      const failedRequests = results.filter((result) => result.status === 'rejected')
+      if (failedRequests.length > 0) {
+        console.error('Failed to load additional user data', failedRequests)
```

Comment:
This will only fail for users who never logged in, right? Since that's expected, shouldn't we avoid the console noise? Maybe reducing the log level to `debug` instead of `error` is an option

Reply:
sounds good, just felt bad to not have any error handling